jupyter-book/multi_omics/resources/celltype_markers.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Can we replace this notebook with a couple of json files? Makes this more easily reusable for other people and it's weird to store this in such a way.</p><p>JSON files can still be easily read into Python dictionaries.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>